src/CodeFormatter/Program.cs

```diff
@@ -18,46 +20,43 @@ namespace CodeFormatter
 {
     internal static class Program
     {
-        private static Assembly[] s_defaultCompositionAssemblies = 
+        private static Assembly[] s_defaultCompositionAssemblies =
                                         new Assembly[] {
                                             typeof(FormattingEngine).Assembly,
                                             typeof(OptimizeNamespaceImportsAnalyzer).Assembly
                                         };
 
         private static int Main(string[] args)
+        {            
+            return Parser.Default.ParseArguments<ListOptions, FormatOptions>(args)
+              .Return(
+                (ListOptions listOptions) => RunListCommandAndReturnExitCode(listOptions),
```

Comment:
Is the reason for including "AndReturnExitCode" in the method names to alert the reader that CommandLine will call `exit` with the return code from this method? I'm not usually a fan of "And" in method names because it suggests the method is doing more than one thing and should be refactored, but I guess it's ok here.